daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md

```diff
@@ -216,11 +216,11 @@ Invoke-RestMethod -Uri 'http://localhost:3601/v1.0/configuration/configstore?key
 {{< /tabs >}}
 
 
-### Subscribe configuration items
+### Subscribe to configuration item updates
 
-Below are the Code examples that leverage Dapr SDKs to subscribe to keys `[orderId1,orderId2]` using `configstore` store. 
+Below are code examples that leverage SDKs to subscribe to keys `[orderId1, orderId2]` using `configstore` store component. 
 
-{{< tabs "Dotnet Extension" "Dotnet Client" Python Go>}}
+{{< tabs ".NET", "ASP.NET Core", Java, Python, Go, Javascript>}}
```

Comment:
Looks like you have .NET and ASP.NET core in the wrong order in the code below. Can you check. 
Also for all these tabs setting, can you check that there is only spaces between them, unless more than one word and then need quotes

Reply:
Resolved